gui/system/models.py

```diff
@@ -132,6 +132,17 @@ class Settings(Model):
         default="America/Los_Angeles",
         verbose_name=_("Timezone")
     )
+    stg_sysloglevel = models.CharField(
+        max_length=120,
+        choices=choices.SYS_LOG_LEVEL,
+        default="f_info",
+        verbose_name=_("Syslog level"),
+        help_text=_("Specifies which messages will be logged by "
+                    "server. INFO and VERBOSE log transactions that "
+                    "server performs on behalf of the client. "
+                    "f_is_debug specify higher levels of debugging output. "
+                    "The default is f_info."),
+    )
```

Comment:
Just to be sure here. Ar eyou sure the default syslog level (when no level is specified at all) is f_info?
Do you have any link to back it up? Because I would think the default is f_notice but I am not sure.